internal/policies/privilege/privilege.go

```diff
@@ -43,24 +45,49 @@ import (
 
 	We are modifying 2 files:
 	- one for sudo, named 99-adsys-privilege-enforcement in sudoers.d
-	- one under 99-adsys-privilege-enforcement.conf for policykit
+	- one under 00-adsys-privilege-enforcement.rules for policykit
 
 	Both are installed under respective /etc directories.
 */
 
-const adsysBaseConfName = "99-adsys-privilege-enforcement"
+const adsysBaseSudoersName = "99-adsys-privilege-enforcement"
```

Comment:
I purposedly left it separated since it relates to the sudoers file and all of the below refer to Polkit. But I can join them if you think it makes more sense.